backstage.yaml

```diff
@@ -1,23 +1,23 @@
 apiVersion: backstage.io/v1alpha1
-kind: Component
+kind: website
```

Comment:
I believe "website" is not a valid option here. See https://backstage.io/docs/features/software-catalog/system-model

Reply:
Jeg tror _type_ kan være "website", men ikke _kind _ . Se linje 18, under spec.